### PR TITLE
Allow defining term_size via PUDB_TERM_SIZE env var

### PR DESCRIPTION
**Pull request by max-arnold**
Implement #628

### CODE REVIEW

File: doc/starting.rst [hunk 1]
```diff
@@ -76,6 +76,10 @@ enabling you to connect via ``telnet``, use the following code::
     from pudb.remote import set_trace
     set_trace(term_size=(80, 24))
 
+The terminal size can be defined via the environment variable as well::
+
+    export PUDB_TERM_SIZE=80x24
+
```
Comment (inducer) [inline doc/starting.rst:82]: Please document which takes precedence in the various cases.

Comment (max-arnold) [reply doc/starting.rst:82]: Done!

In your opinion, does it make sense to support something like `PUDB_RDB_REVERSE` for the sake of completeness?

Comment (inducer) [reply doc/starting.rst:82]: What would the use case be? I.e. in what situation would you be unable to initiate "reverse" debugging in the existing ways?

Comment (max-arnold) [reply doc/starting.rst:82]: The main use-case is using the generic `breakpoint()` to invoke the debugger. Then the debugger parameters have to be defined as env vars (`PYTHONBREAKPOINT`, etc)

Comment (inducer) [reply doc/starting.rst:82]: Not sure I'm sold. If at all, let's do that in a separate PR.

File: pudb/remote.py [hunk 2]
```diff
@@ -139,8 +141,10 @@ def __init__(
 
         if term_size is None:
             try:
-                s = os.get_terminal_size()
-                term_size = (s.columns, s.lines)
+                term_size = tuple(map(int, PUDB_TERM_SIZE.split("x")))
+                if len(term_size) != 2:
```
Comment (inducer) [inline pudb/remote.py:145]: Didn't spot this yesterday: I don't like that this is silently ignored if the format isn't right. It'd be better to raise. (Also above.)

Comment (max-arnold) [reply pudb/remote.py:144]: Done. Now if the `PDB_TERM_SIZE` value is invalid, one of the following exceptions will be raised:

- `ValueError: invalid literal for int() with base 10`
- `ValueError: PUDB_TERM_SIZE should have two dimensions`

**Comment by inducer** [comment]
LGTM, thanks!

**Comment by max-arnold** [comment]
Any chances to see it released soon?

**Pull request merged**